src/HealthChecks.AzureServiceBus/AzureServiceBusSubscriptionHealthCheck.cs

```diff
@@ -61,5 +67,10 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
                 return new HealthCheckResult(context.Registration.FailureStatus, exception: ex);
             }
         }
+
+        protected override string GetConnectionKey()
```

Comment:
can be inlined

```c#
protected override string GetConnectionKey() => $"{Prefix}_{_topicName}_{_subscriptionName}";
```

Could even be a property over a method as it has no side effects?

Reply:
done